rules/linux/process_creation/proc_creation_lnx_browser_information_discovery.yaml

```diff
@@ -0,0 +1,37 @@
+title: Browser Information Discovery - Linux
+id: 53f9703e-abc5-4eff-945e-2ab75c206c63
+status: experimental
+description: Adversaries may enumerate information about browsers to learn more about compromised environments.Data saved by browsers (such as bookmarks, accounts, and browsing history) may reveal a variety of personal information about users (e.g., banking sites, relationships/interests, social media) and details about internal network resources like servers, tools, dashboards, or other infrastructure.
+references:
+    - https://medium.com/@jsaxena017/web-browser-forensics-part-2-firefox-browser-3dc6ef104607
+    - https://renenyffenegger.ch/notes/development/web/browser/Firefox/profile-folder/index
+    - https://medium.com/@s12deff/steal-firefox-passwords-from-windows-linux-9d9a87906c7d
+author: CheraghiMilad
+date: 2024-12-08
+tags:
+    - attack.discovery
+    - attack.t1217
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    selection:
+        Image|endswith:
```

Comment:
Is it easier to exclude valid processes instead of a list of possible editors ?
```yaml
    filter_browser:
        Image|endswith: '/firefox'
condition: selection and not 1 of filter_*
```

Reply:
This is a process creation event, and browsers wont call these via CLI but access them so this suggestion doesn't make sense.